tyrian/js/src/main/scala/tyrian/TyrianAppF.scala

```diff
@@ -12,6 +12,8 @@ import scala.scalajs.js.annotation._
   */
 trait TyrianAppF[F[_]: Async, Msg, Model]:
 
+  def MaxConcurrentMessages: Int = 1024
```

Comment:
Curious as of how to tweak this value for a real-world appliaction.

Reply:
Yeah I was wondering about that. If you know about it you can just override it, but I don't normally do overrides. Indigo requires you to provide a config object as part of the `init` function, I could do that again even though it's just one field (for now).

In practice 1024 is _massive_. I'd expect most apps to rarely even produce 10 at a time.

Reply:
Decided to leave it for now, but I've renamed it to `MaxConcurrentTasks` and added scaladocs explaining that it can be overridden. We'll see how it behaves in the wild and revisit as necessary.